docs/sources/deploy-kubernetes/helm.md

```diff
@@ -128,9 +128,13 @@ Use a custom namespace so that you don't have to overwrite the default namespace
      --set env.GF_AUTH_ANONYMOUS_ORG_ROLE=Admin \
      --set env.GF_DIAGNOSTICS_PROFILING_ENABLED=true \
      --set env.GF_DIAGNOSTICS_PROFILING_ADDR=0.0.0.0 \
-     --set env.GF_DIAGNOSTICS_PROFILING_PORT=6060 \
-     --set-string 'podAnnotations.pyroscope\.grafana\.com/scrape=true' \
-     --set-string 'podAnnotations.pyroscope\.grafana\.com/port=6060'
+     --set env.GF_DIAGNOSTICS_PROFILING_PORT=9094 \
+     --set-string 'podAnnotations.profiles\.grafana\.com/cpu\.scrape=true' \
+     --set-string 'podAnnotations.profiles\.grafana\.com/cpu\.port=9094' \
+     --set-string 'podAnnotations.profiles\.grafana\.com/memory\.scrape=true' \
+     --set-string 'podAnnotations.profiles\.grafana\.com/memory\.port=9094' \
+     --set-string 'podAnnotations.profiles\.grafana\.com/goroutine\.scrape=true' \
+     --set-string 'podAnnotations.profiles\.grafana\.com/goroutine\.port=9094'
```

Comment:
Nit: Instead of reusing the gossip-tcp port, do you think leveraging `extraExposePorts` from Grafana helm chart would work to properly expose the profiling port? I didn't try, but I believe it could be added via Helm command as:
```yaml
--set extraExposePorts[0].name=profiling,extraExposePorts[0].port=6060,extraExposePorts[0].targetPort=6060
```

Reply:
I am afraid that won't do it. extraExposePorts works on service level, we would need it on the container around here: https://github.com/grafana/helm-charts/blob/897c25093da0572716a7cd441e8d3c3dba064b98/charts/grafana/templates/_pod.tpl#L1081

I am planning of submitting something to that chart repo, but that will be a while until it comes through...

Reply:
There we go https://github.com/grafana/helm-charts/pull/3472